css/modal.css

```diff
@@ -0,0 +1,141 @@
+
+.modal {
+  display: flex;
+  flex-direction: column;
+  gap: 0.4rem;
+  overflow-y: auto;
+  width: 90%;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  left: 5%;
+  margin: 10% 1% 10% 0;
+  background-color: #fff;
+  border-radius: 16px;
+  padding: 16px;
+  z-index: 2;
+}
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.4);
+  z-index: 1;
+  backdrop-filter: blur(15px);
+}
+
+.modal .description_wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  margin-top: 30px;
+}
+
+.description_wrapper p.description {
+  color: #344563;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 32px;
+}
+
+.description_wrapper .tags-button_wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding-bottom: 1rem;
+}
+
+.live-source-container {
+  display: flex;
+  gap: 10px;
+}
+
+.live-source-container a {
+  border: 1px solid #6070ff;
+  text-decoration: none;
+  border-radius: 8px;
+  display: flex;
+  flex-wrap: nowrap;
+  padding: 5px 8px;
+  align-items: center;
+  gap: 10px;
+}
+
+.live-source-container a.button:hover {
+  background: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  border-radius: 8px;
+  color: #fff;
+}
+
+.modal .description_wrapper .tags {
+  display: flex;
+  gap: 10px;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  flex-wrap: wrap;
+}
+
+.modal .description_wrapper .tags li {
+  display: flex;
+  align-items: flex-start;
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.modal .modal-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.modal-header a {
+  align-self: flex-end;
+  font-size: 50px;
+  margin-left: 50px;
+  color: rgb(0, 0, 0);
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.modal .snapshoot.desktop {
+  object-fit: cover;
+  height: auto;
+  width: 100%;
+  max-width: 100%;
+}
+
+.hide {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .modal {
+    width: 80%;
+    height: calc(100vh - 40px);
+    left: 10%;
+    padding: 1rem 1em 4em 1rem;
+    margin: 2% 5% 2% 0;
+  }
+
+  .modal .description_wrapper {
+    flex-direction: row;
+  }
+
+  .description_wrapper p.description {
+    width: 90%;
+  }
+
+  .live-source-container a {
+    padding: 6px 6px;
+  }
+}
```

Comment:
- [ ] You did a fantastic job styling your popup:+1: :1st_place_medal:, but as a project requirement you should make sure that the popup is responsive, which means it should fit mobile and desktop sizes properly.
  - I've noticed that you are using an `overflow` to allow the scroll :+1:, but to match with the Figma you shouldn't use scroll inside the popup. 
  - Also kindly don't forget to align your popup to the middle of your page as is shown in Figma. You can check it below.

**Your design**

![Screenshot from 2023-02-02 07-25-50](https://user-images.githubusercontent.com/105117832/216327175-7c4d060d-053a-487e-86c6-f81cff4b5c49.png)

![Screenshot from 2023-02-02 07-29-43](https://user-images.githubusercontent.com/105117832/216327439-0e4637a8-9d25-4999-a24a-2f64324a906a.png)

**Figma**

![Screenshot from 2023-02-02 06-35-29](https://user-images.githubusercontent.com/105117832/216327549-73fff021-5a00-48fc-9b01-1ab7ef0a0218.png)

![Screenshot from 2023-02-02 06-53-58](https://user-images.githubusercontent.com/105117832/216327620-cf1cb297-bfa7-4787-a2b9-c2abefbfaa79.png)

Reply:
Hi @ZewdieMc, yes it will be hidden but you should adjust the height of it to fit the screen size 👍

Reply:
You can use [vh](https://www.w3schools.com/cssref/tryit.php?filename=trycss_unit_vh) to fix that issue